pkg/loop/internal/reporting.go

```diff
@@ -56,9 +62,12 @@ func (r *reportingPluginFactoryClient) NewReportingPlugin(config libocr.Reportin
 	}
 	cc, err := r.brokerExt.dial(reply.ReportingPluginID)
 	if err != nil {
+		if saveErr := r.errorLog.SaveError(ctx, err.Error()); saveErr != nil {
```

Comment:
I'd like to extract this into a utility function so we can save ourselves the duplication, but I wonder which would be the best place to put that function. It would be something like `TrySaveError(ctx, errorLog, err) error`

Reply:
cc: @jmank88 @cedric-cordenier

Reply:
Can we slow down and frame the problem we are trying to solve here first?
All logs and errors are already returned to the host, so this seems redundant in principle.

Reply:
Based on @cedric-cordenier's request, we would like these errors to popup in the UI by saving them when calling `errorLog`. I wonder if we can just modify the line in which we call `NewLOOPPService` on the node, and any error returned just save it in the error log instead. This would make them appear in the UI, right?

Reply:
An "error" log is a solution to a problem. What is the problem?

Reply:
If we _must_ act on every error for some reason, then we should use grpc interceptors to do it in just one place.